### PR TITLE
Remove legacy 'IsBlocking' internal flag from code actions

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/61122

### CODE REVIEW

**Review by mavasani — APPROVED** [review]

**Comment by mavasani** [comment]
@CyrusNajmabadi Seems like a legitimate build failure on the PR.

**Comment by CyrusNajmabadi** [comment]
I've looked through latest typescript-main, and there seems to be no more usages of these types/symbols.  Have also reached out to @zkat and @MariaSolOs about who have confirmed this is currently no longer used by ts.

File: src/Workspaces/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptCodeFixContextExtensions.cs [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-        public static bool IsBlocking(this CodeFixContext context)
-#pragma warning disable CS0612 // Type or member is obsolete
-            => ((ITypeScriptCodeFixContext)context).IsBlocking;
-#pragma warning restore
-
-        public static bool IsBlocking(this CodeRefactoringContext context)
-#pragma warning disable CS0612 // Type or member is obsolete
-            => ((ITypeScriptCodeRefactoringContext)context).IsBlocking;
-#pragma warning restore
-    }
```
Comment (sharwell) [inline src/Workspaces/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptCodeFixContextExtensions.cs:21]: 💡 These could be updated to just return `false` in all cases, without any API break risk.

Comment (CyrusNajmabadi) [reply src/Workspaces/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptCodeFixContextExtensions.cs:21]: these methods aren't called by TS.

**Pull request merged**